terraform/variables.tf

```diff
@@ -375,13 +375,13 @@ variable "github_metrics_dashboard" {
 variable "artifacts_job_name" {
   description = "The name to give the artifacts Cloud Run job"
   type        = string
-  default     = "artifacts_job"
+  default     = "artifacts-job"
```

Comment:
Should we add a regex validation to prevent future errors here?

Reply:
good idea. done.